metricbeat/internal/metrics/opt.go

```diff
@@ -40,6 +42,27 @@ func OptUintWith(i uint64) OptUint {
 	}
 }
 
+// None marks the Uint as not having a value.
+func (opt *OptUint) None() {
```

Comment:
We really need this? What's wrong with just assigning `x = NewUintNone()` in that case.